test/config.test1.yaml

```diff
@@ -26,8 +26,8 @@ snapshots:
 enable:
   prepare_links_p_nom: false
   retrieve_databundle: true
-  build_cutout: false
-  retrieve_cutout: true
+  build_cutout: true
+  retrieve_cutout: false
```

Comment:
cool, is that something that consumes time while testing?

In general the speedups of the build_renewables_profiles should greatly speedup the CI!

Reply:
Yes, that's temporarily for the CI. Actually, once it's in the 'short term memory' of (in this case) my ecmwf account, retrieving data is very fast. Holds for roughly 2 days (from what I experienced).